.gitignore

```diff
@@ -3,6 +3,7 @@
 /public/storage
 /storage/*.key
 /vendor
+/htdocs
```

Comment:
need this?

Reply:
Yes, my local setup requires it. It doesn't really hurt.